src/executor/admin/KillQueryExecutor.cpp

```diff
@@ -141,5 +146,29 @@ Status KillQueryExecutor::verifyTheQueriesByMetaInfo(
     }
     return Status::OK();
 }
+
+void KillQueryExecutor::findKillQueriesViaMeta(
```

Comment:
Change `findKillQueriesViaMeta` -> `findKilledQueriesViaMeta` may be more appropriate.